azure-pipelines.yml

```diff
@@ -1,162 +1,888 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# The Task 'PublishBuildArtifacts@1' have been converted to an output named 'Publish bundle analysis' in the templateContext section.
```

Comment:
Left this in as helpful pointers for code reviewers. Should remove before checkin